time_tracker_api/projects/projects_model.py

```diff
@@ -8,7 +8,7 @@
 from time_tracker_api.database import CRUDDao, APICosmosDBDao
 from typing import List, Callable
 from commons.data_access_layer.database import EventContext
-from utils.repository import convert_list_to_tuple_string, create_sql_in_condition
```

Comment:
`convert_list_to_tuple_string` is no longer to be used?

Reply:
It is still used but in CosmosDBQueryBuilder

Reply:
That's right, CosmosDBQueryBuilder uses convert_list_to_tuple_string. It is no longer necessary to import it into projects_model